### PR TITLE
fix(config): MCOHome configuration changes

### PR DESCRIPTION
**Pull request by kanebullen**
Hopefully have done properly this time.  Various changes to config files

closes: #2151

### CODE REVIEW

**Comment by kanebullen** [comment]
@blhoward2 Hopefully these are all looking okay.     

I notice the check for "test and release / lint" is failing with some non-specific formatting issues - can't seem to find any information on how to address?

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 1]
```diff
@@ -83,7 +139,7 @@
 			"allowManualEntry": true
 		},
 		"7": {
-			"label": "Single dimming step time(Manual)",
+			"label": "Single dimming step time (Manual) 10ms steps",
```
[the remark this answers is not in the document]
Comment (AlCalzone) [reply packages/config/config/devices/0x015f/mh-p220.json]: Some of the following params are missing the unit too, but I can't select them in the review.

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 2]
```diff
@@ -182,7 +258,29 @@
...
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Wire with Roller Blinds Switch",
+					"value": 0
+				},
+				{
+					"label": "Enable Scene Activation and send basic set on/off to AG4",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220.json]:
```suggestion
					"label": "Enable Scene Activation and send Basic Set On/Off to group 4",
```

File: packages/config/config/devices/0x015f/mh-s212.json [hunk 3]
```diff
@@ -21,5 +25,188 @@
 			"maxNodes": 5,
 			"isLifeline": true
 		}
+	},
+	"supportsZWavePlus": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s212.json]: We don't use `supportsZWavePlus` anymore

File: packages/config/config/devices/0x015f/mh-s212.json [hunk 4]
```diff
@@ -21,5 +25,188 @@
...
+					"value": 1
+				},
+				{
+					"label": "Report off only",
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Switch 1 - Off Timer",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s212.json:115]: This label doesn't seem to match the options.

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+					"value": 0
+				},
+				{
+					"label": "When turned on, auto off after X seconds",
+					"value": 1
+				},
+				{
+					"label": "When turned off, auto on after X seconds",
+					"value": 2
+				},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s511.json]: What's X?

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+				{
+					"label": "Hold more than 3 seconds, turns on, when released, turns off",
+					"value": 3
+				},
+				{
+					"label": "Single click to switch on/off, Hold more than 3 seconds will turn on, when released, turns off",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s511.json]: These labels here are very long, can we shorten them?

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Basic Set Received, Key 1 responds.  Basic Get Received, Key 1 sends Basic Report.  Key 1 WILL NOT send unsolicited Basic Report (No Endpoint) to Life Line Association",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s511.json]: These labels are even worse :D
You could put part of this into the parameter description and reference them instead

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks pretty good in general. Some labels should be improved/shortened though. I've commented S511, but these comments also apply to the other files, especially S512/513

File: packages/config/config/devices/0x015f/mh-s212.json (hunk 3 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s212.json]: OK have removed

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 8]
```diff
@@ -22,45 +26,97 @@
...
+					"value": 0
+				},
+				{
+					"label": "Toggle (2-State Switches)",
+					"value": 1
+				},
+				{
+					"label": "Roller Blind Switch - S1 Dim Up, S2 Dim Down",
+					"value": 1
+				}
```
[the remark this answers is not in the document]
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-p220.json]: Addressed

File: packages/config/config/devices/0x015f/mh-s212.json (hunk 4 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s212.json:115]: have fixed

File: packages/config/config/devices/0x015f/mh-p220.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-p220.json]: Addressed

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 5 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json]: X is Parameter 6 (Key On Duration), so I've referenced that in my change

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 6 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json]: Have shorted these a bit (hopefully enough)

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 7 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json]: I agree they are very long (they are taken from the manual).  I've reduced down a bunch, and hopefully all ok.

**Comment by kanebullen** [comment]
@zwave-js-bot fix lint

**Comment by kanebullen** [comment]
@AlCalzone hopefully all ok to go now

File: packages/config/config/devices/0x015f/mh-s212.json [hunk 9]
```diff
@@ -117,17 +116,17 @@
...
 				{
-					"label": "Toggle Switch without specified on/off position",
+					"label": "Forbid Manual Operation",
 					"value": 0
 				},
 				{
-					"label": "Toggle Switch with specified on/off position",
+					"label": "Allow Manual Operation",
 					"value": 1
 				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-s212.json]: I don't understand what these options have to do with the label `"Switch 1 - Off Timer"`

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
@blhoward2 what do you think? Some of these labels are long, but I don't see how we can shorten them.

File: packages/config/config/devices/0x015f/mh-s212.json (hunk 9 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s212.json]: Yeah, I'm not sure to be honest.   I'm wondering whether it should just be "Disable" and "Enable", which will enable at automatic off timer after pressing Switch 1.  (With the following parameter 6 (Timing Function) specifying the amount of time to turn off?

image

Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s212.json]: I'm thinking that approach makes sense, so I've updated to suit.

**Review by AlCalzone — DISMISSED** [review]

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 10]
```diff
@@ -1,10 +1,10 @@
...
-// MCOHome Dimmer Switch
+// McoHome Technology Co., Ltd. MH-DT311 / MH-DT411 / MH-DT511
+// Dimmer Switch
 {
 	"manufacturer": "McoHome Technology Co., Ltd.",
 	"manufacturerId": "0x015f",
-	"label": "MH-DT311/MH-DT411",
-	"description": "MCOHome Dimmer Switch",
+	"label": "MH-DT311 / MH-DT411 / MH-DT511",
+	"description": "Dimmer Switch",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-dt311_411.json:7]: Please carry all of my changes to this file through to other files.

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 11]
```diff
@@ -168,9 +173,10 @@
 			"allowManualEntry": true
 		},
 		"9": {
-			"label": "Single Dimming Step Time (Auto)",
-			"unit": "10 ms",
+			"label": "Dimming Step Time (Auto)",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-dt311_411.json:175]: Does the manual say Auto here, or Z-Wave? Typically the dim steps are defined for manual, z-wave, and all on/off.

File: packages/config/config/devices/0x015f/mh-dt311_411.json [hunk 12]
```diff
@@ -449,19 +459,33 @@
...
 					"value": 0
 				},
 				{
-					"label": "Enabled",
+					"label": "Enable",
 					"value": 1
 				}
 			]
 		},
+		"35": {
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-dt311_411.json:467]: We're making changes to preserve the order of the file in the UI. To me it makes sense for the delay to come after the enable/disable.

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 13]
```diff
@@ -13,57 +13,112 @@
...
+					"value": 255
+				}
+			]
 		},
 		"5": {
-			"label": "Minimum brightness level",
+			"label": "Minimum Brightness",
 			"valueSize": 1,
 			"minValue": 1,
 			"maxValue": 98,
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json:124]: Is 98 correct here? Not 99?

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 14]
```diff
@@ -182,7 +260,29 @@
...
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Wire with Roller Blinds Switch",
+					"value": 0
+				},
+				{
+					"label": "Enable Scene Activation and send basic set on/off to AG4",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: I think this needs to be even shorter. How about: "Scene activation; Control Association Group 4"

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 15]
```diff
@@ -182,7 +260,29 @@
...
+					"value": 1
+				},
+				{
+					"label": "Scene Activation only",
+					"value": 2
+				},
+				{
+					"label": "Send Basic On/Off only",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Change to Basic CC Set

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 16]
```diff
@@ -182,7 +260,29 @@
...
+				{
+					"label": "Wire with Roller Blinds Switch",
+					"value": 0
+				},
+				{
+					"label": "Enable Scene Activation and send basic set on/off to AG4",
+					"value": 1
+				},
+				{
+					"label": "Scene Activation only",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Scene Activation. "Only" is superfluous.

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 17]
```diff
@@ -182,7 +260,29 @@
...
+					"value": 2
+				},
+				{
+					"label": "Send Basic On/Off only",
+					"value": 3
+				},
+				{
+					"label": "Disable S2 Function",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Just make this "Disable"

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 18]
```diff
@@ -182,7 +260,29 @@
...
+					"value": 255
+				}
+			]
 		},
 		"18": {
 			"label": "Operation report Switch2 to Association Group 4",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Change to "Association Group 4 Commands"

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 19]
```diff
@@ -193,7 +293,25 @@
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send Both Basic On and Off",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Basic CC Set (On/Off)

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 20]
```diff
@@ -193,7 +293,25 @@
...
+					"value": 0
+				},
+				{
+					"label": "Send Basic On",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Basic CC Set (On)

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 21]
```diff
@@ -193,7 +293,25 @@
...
+					"value": 1
+				},
+				{
+					"label": "Send Basic Off",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Basic CC Set (Off)

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 22]
```diff
@@ -193,7 +293,25 @@
...
+					"value": 2
+				},
+				{
+					"label": "No Report",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Change to "Disable"

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 23]
```diff
@@ -227,6 +345,27 @@
...
+			"minValue": 0,
+			"maxValue": 85,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: "Normal operation"

File: packages/config/config/devices/0x015f/mh-p220.json [hunk 24]
```diff
@@ -227,6 +345,27 @@
...
+					"value": 0
+				},
+				{
+					"label": "Factory Reset",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-p220.json]: Factory reset. Option labels should be sentence case per the style guide.

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 25]
```diff
@@ -0,0 +1,357 @@
...
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Key1 Responds to Basic Set and Get. No unsolicited report to Lifeline",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Well, these are messy parameters. How about "Key1 (Basic CC Set/Get); Reporting disabled"

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 26]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 0
+				},
+				{
+					"label": "Key1 Responds to Basic Set and Get. Key1 WILL send unsolicited report to Lifeline",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Key1 (Basic CC Set/Get); Reporting enabled

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 27]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 1
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. No unsolicited report to Lifeline",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: All keys (Basic CC Set only); Reporting disabled

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 28]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 2
+				},
+				{
+					"label": "All Keys respond to Basic Set. Basic Get ignored. All keys send unsolicited report to Lifeline",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: All keys (Basic CC Set only); Reporting enabled

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 29]
```diff
@@ -0,0 +1,357 @@
...
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Single click to switch on/off",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Single click (On/Off)

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 30]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 0
+				},
+				{
+					"label": "When turned on, auto off after 'Key On Duration' seconds",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Auto off after 'Key on Duration' seconds

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 31]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 1
+				},
+				{
+					"label": "When turned off, auto on after 'Key On Duration' seconds",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Auto on after 'Key on Duration' seconds

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 32]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 2
+				},
+				{
+					"label": "Hold >3secs turns on, when released, turns off",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Held 3 seconds -> On; Release -> Off

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 33]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 3
+				},
+				{
+					"label": "Single click for on/off, Hold >3secs on, release turns off",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Single click (On/Off); Held 3 seconds -> On; Release -> Off

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 34]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 4
+				},
+				{
+					"label": "Momentary switch: hold key turns on, off once released",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Etc.

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 35]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 5
+				},
+				{
+					"label": "Hold >3secs turns on, Hold >3secs turns off",
+					"value": 6
+				}
+			]
+		},
+		"14": {
+			"label": "Disable Local Control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Local Control

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 36]
```diff
@@ -0,0 +1,357 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for local control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Enable

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 37]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for local control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Disable

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 38]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 1
+				}
+			]
+		},
+		"15": {
+			"label": "Disable Remote Control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Remote Control

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 39]
```diff
@@ -0,0 +1,357 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "All keys are valid for remote (gateway) control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Enable

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 40]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 0
+				},
+				{
+					"label": "All keys are invalid for remote (gateway) control",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Disable

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 41]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 1
+				}
+			]
+		},
+		"16": {
+			"label": "Scene Respond",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json:241]: This label is confusing. What does it do?

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 42]
```diff
@@ -0,0 +1,357 @@
...
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Scene Respond Disabled",
+					"value": 0
+				},
+				{
+					"label": "Key 1 responds, Keys 2 and 3 do not respond",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Are there only three keys? If so, just make it "Key 1 responds," then "Key 2 responds" for the next option, etc. If only one is listed and other options list only other keys it is implicit that the others do nothing.

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 43]
```diff
@@ -0,0 +1,357 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Scene Activate function disabled",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Scene activation disabled

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 44]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 0
+				},
+				{
+					"label": "Upon clicking Key 1, always activate scene ID1 no matter what the status of Key 1 is",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Key 1 -> Scene ID1 (Any Key 1 relay state)

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 45]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 1
+				},
+				{
+					"label": "Upon clicking Key 1, always activate scene ID1 when Key 1's relay output is open",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Key 1 -> Scene ID 1 (Key 1 relay open)

File: packages/config/config/devices/0x015f/mh-s511.json [hunk 46]
```diff
@@ -0,0 +1,357 @@
...
+					"value": 6
+				},
+				{
+					"label": "Keys 1, 2 and 3 all respond to scene",
+					"value": 7
+				}
+			]
+		},
+		"17": {
+			"label": "Key 1 Scene Activate Mode Setting",
```
Comment (blhoward2) [inline packages/config/config/devices/0x015f/mh-s511.json]: Key 1 Scene Settings

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Please take a look at the changes I pushed and my comments and incorporate throughout. Request another review when complete.

File: packages/config/config/devices/0x015f/mh-dt311_411.json (hunk 11 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-dt311_411.json:175]: Yes the manual does mention "Auto" - seems that the dim steps correspond to three situations:

image

File: packages/config/config/devices/0x015f/mh-p220.json (hunk 13 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-p220.json:124]: Yes, 98 is correct - max brightness must be greater than minimum brightness.  So Minimum is 1-98, Max is 2-99

image

File: packages/config/config/devices/0x015f/mh-dt311_411.json (hunk 10 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-dt311_411.json:7]: Will do.

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 25 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json]: agreed they are messy!

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 41 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json:241]: I actually have no idea - it's straight from the manual.  Would it be better to remove the parameter from the files?

image

File: packages/config/config/devices/0x015f/mh-s511.json (hunk 42 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-s511.json]: That makes sense.    will do

File: packages/config/config/devices/0x015f/mh-dt311_411.json (hunk 12 above)
Comment (kanebullen) [reply packages/config/config/devices/0x015f/mh-dt311_411.json:467]: agreed

**Review by blhoward2 — APPROVED** [review]
Made some additional minor style changes. LGTM.

**Pull request merged**